# Goal: triggers the controller.
int32 rod_arm
int32 surface_arm
float64 position_offset
float64 orientation_goal
float64 force_goal

int32 LEFT_ARM = 0
int32 RIGHT_ARM = 1
---
# Result:

---
# Feedback: information that may be used to debug the controller
geometry_msgs/TwistStamped controller_output
geometry_msgs/Vector3 r1
geometry_msgs/Vector3 r2
geometry_msgs/Vector3 p1
geometry_msgs/Vector3 p2
geometry_msgs/Vector3 pc
geometry_msgs/WrenchStamped surface_wrench
